src/main/js/main_html5.js

```diff
@@ -86,7 +86,16 @@ require("../../../html5-common/js/utils/environment.js");
       var video = $("<video>");
       video.attr("class", "video");
       video.attr("id", domId);
-      video.attr("preload", "none");
+
+      // [PBW-5470] On Safari, when preload is set to 'none' and the user switches to a
+      // different tab while the video is about to auto play, the browser stops playback but
+      // doesn't fire a 'pause' event, which causes the player to get stuck in 'buffering' state.
+      // Setting preload to 'metadata' (or 'auto') allows Safari to auto resume when the tab is refocused.
```

Comment:
I raised this question because in comment you wrote, "Setting preload to 'metadata' (or 'auto') allows Safari ..." - so I thought they are the same.  Do you actually know what is the actual difference between them?